styles/articles.css

```diff
@@ -37,9 +37,163 @@ main video {
 @media (min-width: 700px) {
   main .section > div > :not(div):not(h1),
   main .section > .tags,
-  main .columns, main .columns.contained {
+  main .columns, main .columns.contained,
+  main .blog-wrapper-content > div {
     max-width: 800px;
     margin-left: auto;
     margin-right: auto;
   }
 }
+
+@media (min-width: 900px) {
+  main .section:first-of-type {
+    margin-top: var(--global-height-nav);
+  } 
+}
+
+/* Article v3 related styles */
+main .article-hero-banner .text p {
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+@media (max-width: 600px) {
+  main .article-hero-banner .text p {
+    display: none;
+  }
+}
+
+/* tags */
+main .tags {
+  max-width: none;
+  margin-top: var(--spacing-m);
+}
+
+main .tags a {
+  font-weight: 700;
+  letter-spacing: normal;
+  padding: 12px;
+  margin: 0 12px 12px 0;
+}
+
+@media (min-width: 900px) {
+  main .tags {
+    margin-top: var(--spacing-s);
+  }
+}
+
+/* Blog Styles (sidebar + content) */
+main .article-hero-category-link {
+  display: inline-block;
+  color: var(--color-white);
+}
+
+main .light .article-hero-category-link {
+  color: var(--text-color);
+}
+
+.blog-wrapper {
+  display: flex;
+  flex-direction: column-reverse;
+  gap: var(--spacing-xs);
+  margin: 0 auto;
+  max-width: var(--grid-container-width);
+  padding-bottom: var(--spacing-m);
+}
+
+.blog-wrapper-sidebar {
+  display: none;
+}
+
+.blog-wrapper-sidebar .tags {
+  max-width: none;
+}
+
+main .blog-wrapper-sidebar .tags a {
```

Comment:
Why are these styles sometimes scoped to `main` and sometimes not? Should keep it consistent and lean concise if possible.